CityTankGame.java

```diff
@@ -34,7 +34,7 @@ public static void main(String[] args)
 
       //Get user command
       System.out.println("Control your tank by typing a character and press Enter. ");
-      System.out.println("W: up, A: left, S: right, Z: down, Q: quit the game.");
+      System.out.println("W: up, A: left, D: right, S: down, Q: quit the game.");
```

Comment:
I don't think this needs to be changed. It is valid to use a non-WASD control.

Reply:
I have reverted it.